packages/manager/apps/pci-users/src/index.css

```diff
@@ -5,3 +5,11 @@
 .application {
   @apply px-12 pt-8;
 }
+
+code {
+  padding: 2px 4px !important;
+  font-size: 90% !important;
+  color: #c7254e !important;
+  background-color: #f9f2f4 !important;
+  border-radius: 6px !important;
+}
```

Comment:
We need to keep this custom CSS because this CSS is used for the code tag in translation file (ex: https://github.com/ovh/manager/blob/feat/pci-users-renew-password/packages/manager/apps/pci-users/public/translations/common/Messages_en_GB.json#L21)